modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -698,91 +642,45 @@ void RenderableFieldlinesSequence::setupProperties() {
 
     if (hasExtras) {
         // Set defaults
-        _pColorQuantity = 0;
-        _pColorQuantityMin = std::to_string(_colorTableRanges[0].x);
-        _pColorQuantityMax = std::to_string(_colorTableRanges[0].y);
-        _pColorTablePath = _colorTablePaths[0];
-
-        _pMaskingQuantity = 0;
-        _pMaskingMin = std::to_string(_maskingRanges[0].x);
-        _pMaskingMax = std::to_string(_maskingRanges[0].y);
+        _colorQuantity = _colorQuantityTemp;
+        _colorQuantityMinMax = _colorTableRanges[_colorQuantity];
+        _colorTablePath = _colorTablePaths[0];
+
+        _maskingQuantity = _maskingQuantityTemp;
+        _maskingMinMax = _maskingRanges[_colorQuantity];
     }
 }
 
 void RenderableFieldlinesSequence::definePropertyCallbackFunctions() {
     // Add Property Callback Functions
     bool hasExtras = (_states[0].nExtraQuantities() > 0);
     if (hasExtras) {
-        _pColorQuantity.onChange([this] {
+        _colorQuantity.onChange([this] {
             _shouldUpdateColorBuffer = true;
-            _pColorQuantityMin = std::to_string(_colorTableRanges[_pColorQuantity].x);
-            _pColorQuantityMax = std::to_string(_colorTableRanges[_pColorQuantity].y);
-            _pColorTablePath = _colorTablePaths[_pColorQuantity];
-        });
-
-        _pColorTablePath.onChange([this] {
-            _transferFunction->setPath(_pColorTablePath);
-            _colorTablePaths[_pColorQuantity] = _pColorTablePath;
+            _colorQuantityMinMax = _colorTableRanges[_colorQuantity];
+            _colorTablePath = _colorTablePaths[_colorQuantity];
         });
 
-        _pColorQuantityMin.onChange([this] {
-            const float f = stringToFloat(
-                _pColorQuantityMin,
-                _colorTableRanges[_pColorQuantity].x
-            );
-            _pColorQuantityMin = std::to_string(f);
-            _colorTableRanges[_pColorQuantity].x = f;
+        _colorTablePath.onChange([this] {
```

Comment:
Missing empty parameter paranthesis:
`[this]() {`

Same below

Reply:
Otherwise it's a bit harder for human readers to parse this as a lambda expression